metabase/metabase
# Improve slack settings messages

Fixes https://github.com/metabase/metabase/issues/20936

How to test:
- Go to Admin -> Settings -> Slack and complete the setup
- Make sure the title and description for the slack channel name form field has been changed

<img width="671" alt="Screenshot 2022-09-13 at 13 19 19" src="https://user-images.githubusercontent.com/8542534/189877103-f0a7f5c4-ed59-46e1-bd65-ebef582b30c1.png">
<img width="692" alt="Screenshot 2022-09-13 at 13 20 36" src="https://user-images.githubusercontent.com/8542534/189877263-0a3db9a1-12d0-42d2-b042-208ad301b96f.png">
